File: api/code-samples/python/klay/account/sign.py
import json
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

address = "0xa94f5374fce5edbc8e2a8697c15331677e6ebf0b"
message = "0xdeadbeaf"

sdk = OpenSDK(host)
klay_response = sdk.klay.sign(address, message)

print(json.loads(klay_response.response.data))

File: web3rpc/rpc-specs/api/code-samples/javascript/klay/account/sign.js
const OpenSdk = require("opensdk-javascript");

(() => {
    const sdk = new OpenSdk(new OpenSdk.ApiClient("https://api.baobab.klaytn.net:8651"));


    const address = '0x487f2dfef230c2120b8cc55c5087b103146536ec'
    const message = '0xdeadbeaf'

    sdk.klay.sign(address, message, {}, (err, data, response) => {
        console.log(data);
    });
}
)()